fluentd/lib/parser_json_or_crio/parser_json_or_crio.gemspec

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+# can override for testing
+FLUENTD_VERSION = ENV['FLUENTD_VERSION'] || "0.12.0"
+
+Gem::Specification.new do |gem|
+  gem.name          = "parser_json_or_crio"
+  gem.version       = "0.0.1"
+  gem.authors       = ["Rich Megginson"]
+  gem.email         = ["rmeggins@redhat.com"]
+  gem.description   = %q{Parser plugin for in_tail to determine if line is json or crio format and parse accordingly}
+  gem.summary       = %q{Parser plugin for in_tail to determine if line is j    son or crio format and parse accordingly}
+  gem.homepage      = "https://github.com/openshift/origin-aggregated-logging"
+  gem.license       = "Apache-2.0"
+
+  gem.files         = `git ls-files`.split($/)
```

Comment:
Prefer Dir glob'ing over exec.  Though I guess this captures the files which are under source control?